ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotStyleFormatEnums.cs

```diff
@@ -5,7 +5,6 @@ namespace ClosedXML.Excel
     [Flags]
     public enum XLPivotStyleFormatElement
     {
-        None = 0,
```

Comment:
Why remove the default value? Since `enum` is a value type, the variable of this type always has value, even before it is assigned explicitly. I rather have `None` than `0` in that case.

Reply:
Because there are only two possible options - Label or Data. The third option can not be processed. If leave None, will need to check and throw an Exception in the case of None. I removed it to eliminate possible errors.

Reply:
Default here should be Data.

Reply:
Dealing with `enum`s you always have to keep in mind that there can be other values, not listed in the enum because somebody can cast an arbitrary value to it (`(XLPivotStyleFormatElement)999`) intentionally or by mistake. And if the code does not accept `None` this should be stated explicitly; removing valud from enum is not enough.

Consider this:
```
struct Test
{
  public XLPivotStyleFormatElement StyleFormatElement {get; set;}
}

var t = new Test();
//t.StyleFormatElement == (XLPivotStyleFormatElement)0 at this point, but XLPivotStyleFormatElement.None or XLPivotStyleFormatElement.NotSet looks nicer.
```

Reply:
If you leave `None`, users will make such calls `AddSubtotalStyle(XLPivotStyleFormatElement.None)`, which will lead to Exception.
I understand that you are trying to adhere to the rules/patterns, but will it be correct to give users a choice of pre-disabled options?
The only place where the property XLPivotStyleFormatElement is declared [here](https://github.com/ClosedXML/ClosedXML/blob/74e5c1bdb1e4263d158e6766ab49e49072578ecb/ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotFormat.cs#L14). In this case, the default is Data.

Reply:
Fair point. Accepted